doc/docstyle.css

```diff
@@ -254,26 +254,24 @@ img {
 
   .avatar-container,
   .badge_container,
-  .alert_container {
-    margin: 2rem 0rem;
-    padding: 0rem 0rem 2rem 0rem;
+  .alert_container,
+  .card_container {
+    margin: 2rem 0;
+    padding-bottom: 2rem ;
   }
 
   .footer img {
     height: 1.5rem;
     margin: 0.5rem;
   }
 
-
-  .code_container{
+  .code_container {
     padding: 2rem;
-    
   }
-  
-  .code_container iframe{
-    width: 390px; height: 200px;
+
+  .code_container iframe {
+    width: 390px;
```

Comment:
you can make width as 100 % and height auto to make it responsive

Reply:
It is still responsive but I'll try this also.